### PR TITLE
Create documentation for iAlarmXR device support

### PR DESCRIPTION
**Pull request by bigmoby**
## Proposed change
This PR extends Home Assistant component core support for iAlarmXR model. The current iAlarm Home Assistant component support is for GuardIP model. Unfortunately the two alarm system differs on authentication flow but the remaining API remains quite the same.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [X] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [X] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands). https://github.com/home-assistant/brands/pull/3353
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase:  https://github.com/home-assistant/core/pull/67817
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [X] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/ialarmxr.markdown [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+title: Antifurto365 iAlarmXR
+description: Instructions on how to integrate iAlarmsXR alarms into Home Assistant.
+ha_category:
+  - Alarm
+ha_iot_class: Local Polling
```
Comment (RyuzakiKK) [inline source/_integrations/ialarmxr.markdown]:
```suggestion
ha_iot_class: Cloud Polling
```

As you mentioned in the integration PR, this uses the iAlarm cloud.

Comment (bigmoby) [reply source/_integrations/ialarmxr.markdown]: Good point! I'll fix also the manifest!

Comment (bigmoby) [reply source/_integrations/ialarmxr.markdown]: Thanks @RyuzakiKK also for docs review! 🙌 Now it's fixed, thanks

**Comment by klaasnicolaas** [comment]
I see a few changes that I don't think belong in this pull request (package-lock file)

**Comment by bigmoby** [comment]
> I see a few changes that I don't think belong in this pull request (package-lock file)

Thank you @klaasnicolaas 👍 I reverted!
Probably a "git add ." after I run the local container
Thank you!

File: source/_integrations/ialarmxr.markdown [hunk 2]
```diff
@@ -0,0 +1,24 @@
+---
+title: Antifurto365 iAlarmXR
+description: Instructions on how to integrate iAlarmsXR alarms into Home Assistant.
+ha_category:
+  - Alarm
+ha_iot_class: Cloud Polling
+ha_config_flow: true
+ha_release: '2021.5'
```
Comment (bdraco) [inline source/_integrations/ialarmxr.markdown]: This version needs to be adjusted

Comment (bigmoby) [reply source/_integrations/ialarmxr.markdown]: Thank you @bdraco 👍

**Review by frenck — CHANGES_REQUESTED** [review]
Seems like something got lost during rebase?

There are unrelated changes in this PR and additionally the page for ialarm_xr is missing?

**Comment by bigmoby** [comment]
> Seems like something got lost during rebase?
> 
> There are unrelated changes in this PR and additionally the page for ialarm_xr is missing?

Uh my fault! 🤦‍♂️  Sorry @frenck I forgot to add the new renamed file! I fixed it

File: source/_integrations/ialarm_xr.markdown [hunk 3]
```diff
@@ -0,0 +1,24 @@
+---
+title: Antifurto365 iAlarmXR
+description: Instructions on how to integrate iAlarmsXR alarms into Home Assistant.
+ha_category:
+  - Alarm
+ha_iot_class: Cloud Polling
+ha_config_flow: true
+ha_release: '2022.5.4'
```
Comment (frenck) [inline source/_integrations/ialarm_xr.markdown]:
```suggestion
ha_release: 2022.6
```

**Review by frenck — CHANGES_REQUESTED** [review]
Version needs a change. (resolved)

**Review by frenck — CHANGES_REQUESTED** [review]
There are unrelated changes in this PR:

CleanShot 2022-05-25 at 14 46 38

**Comment by bigmoby** [comment]
> There are unrelated changes in this PR:
> 
> CleanShot 2022-05-25 at 14 46 38

I don't know why this file is changed every commit I push! I changed it with the current home-assistant:next branch version

**Comment by RyuzakiKK** [comment]
> I don't know why this file is changed every commit I push! I changed it with the current home-assistant:next branch version

There are still changes to the `package-lock.json` file. I guess it didn't work as expected because your branch is behind the current HA `next`, so your PR still includes edits in `package-lock.json`.

**Comment by bigmoby** [comment]
> behind

Yes but...I don't know why 🤔 ...now I merged next branch into feature/ialarmxr branch...mmm

**Comment by RyuzakiKK** [comment]
> Yes but...I don't know why thinking ...now I merged next branch into feature/ialarmxr branch...mmm

I see that you have already rebased. So you can simply download the latest version of the file from https://raw.githubusercontent.com/home-assistant/home-assistant.io/next/package-lock.json and include that. It should cancel out any committed changes so far.

**Comment by bigmoby** [comment]
> > Yes but...I don't know why thinking ...now I merged next branch into feature/ialarmxr branch...mmm
> 
> I see that you have already rebased. So you can simply download the latest version of the file from https://raw.githubusercontent.com/home-assistant/home-assistant.io/next/package-lock.json and include that. It should cancel out any committed changes so far.

Thanks @RyuzakiKK it works 👍

**Review by frenck — APPROVED** [review]
Thanks, @bigmoby 👍

**Comment by bigmoby** [comment]
> Thanks, @bigmoby 👍

To you @frenck 🙌

**Pull request merged**